### PR TITLE
Db 1699 submission file organization

### PR DESCRIPTION
**Pull request by alburde1**
Updating the keys for storing submissions in S3 (use submission_id instead of user_id)

### CODE REVIEW

File: dataactbroker/handlers/fileHandler.py [hunk 1]
```diff
@@ -969,7 +969,7 @@ def build_file_map(self, request_params, file_type_list, response_dict, upload_f
             if file_name:
                 if not self.isLocal:
                     upload_name = "{}/{}".format(
-                        g.user.user_id,
+                        submission.submission_id,
```
Comment (nmonga91) [inline dataactbroker/handlers/fileHandler.py]: This handles updating file paths for an existing submission correct?

Comment (alburde1) [inline dataactbroker/handlers/fileHandler.py]: No, that's what the one-time script I'm working on will do (although if someone uploads a new file in an existing submission this SHOULD work as intended as well)

Comment (nmonga91) [inline dataactbroker/handlers/fileHandler.py]: Yeah the latter was my concern, when a user updates a submission, this'll put the new file in the right place. Looks good to me 👍

**Comment by nmonga91** [comment]
Looks good, merging

**Pull request merged**